kong/plugins/prometheus/exporter.lua

```diff
@@ -336,6 +368,38 @@ local function metric_data()
     enterprise.metric_data()
   end
 
+  -- Hybrid mode status
+  if cp_metrics then
+    -- Cleanup old metrics
+    metrics.data_plane_last_seen:reset()
+    metrics.data_plane_config_hash:reset()
+
+    for data_plane, err in kong.db.clustering_data_planes:each() do
+      if err then
+        kong.log.err("failed to list data planes: ", err)
+        goto next_data_plane
```

Comment:
There are not nested loops, why not `break`?

Reply:
this is essentially a `continue`, since we still process next row if current one is somehow error'ed.